app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -19,6 +19,9 @@ class Api::V0::UserRolesController < Api::V0::ApiController
     group_id = params[:groupId]
     user_id = params[:userId]
 
+    # For post filtering, user & group is needed.
+    active_record = active_record.includes([:user, :group])
```

Comment:
First, `includes` takes varargs, so `.includes(:user, :group)` will work.
Second, if you know that these two are going to be `includes`'d anyways no matter what, then you can just declare `active_record = UserRole.includes(...)` directly at the top of the method above.

Reply:
Oh ya, thanks for that. Implemented.